nodebuilder/settings.go

```diff
@@ -123,13 +122,9 @@ func initializeMetrics(
 		metric.WithReader(metric.NewPeriodicReader(exp, metric.WithTimeout(2*time.Second))),
 		metric.WithResource(resource.NewWithAttributes(
 			semconv.SchemaURL,
-			semconv.ServiceNameKey.String(fmt.Sprintf("Celestia-%s", nodeType.String())),
+			semconv.ServiceNamespaceKey.String(fmt.Sprintf("Celestia-%s", nodeType.String())),
+			semconv.ServiceNameKey.String(fmt.Sprintf("semver-%s", buildInfo.SemanticVersion)),
```

Comment:
Why not the reverse? 
```suggestion
			semconv.ServiceNameKey.String(fmt.Sprintf("Celestia-%s", nodeType.String())),
			semconv.ServiceNamespaceKey.String(fmt.Sprintf("semver-%s", buildInfo.SemanticVersion)),
```

Reply:
it depends on the result you want to get:
the current implementation will look like 
`job="Celestia-Light/semver-v0.9.2-4-gd06913b0-dev"`

Celestia-Light - namespace
v0.9.2-4-gd06913b0-dev - name

With the proposed one: semver-v0.9.2-4-gd06913b0-dev/Celestia-Light"

Reply:
I see